airbyte-webapp/.eslintrc.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
     "plugin:jsx-a11y/recommended",
     "plugin:@airbyte/recommended",
   ],
-  plugins: ["react", "@typescript-eslint", "prettier", "unused-imports", "css-modules", "jsx-a11y", "@airbyte"],
```

Comment:
:information_source: This is with `pnpm` now conflicting with the `react` plugin already being pulled in by `eslint-config-react-app` which we extend above. Since that plugin pulls in the `react` plugin also no need for us to pull it in explicitally.